src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -848,4 +848,13 @@ Do you want to proceed?</value>
   <data name="Xml_Tag_Completion_Command_Handler" xml:space="preserve">
     <value>XML Tag Completion Command Handler</value>
   </data>
+  <data name="Code_cleanup_is_not_configured" xml:space="preserve">
+    <value>Code cleanup is not configured</value>
+  </data>
+  <data name="Configure_it_now" xml:space="preserve">
+    <value>Configure it now</value>
+  </data>
+  <data name="Donot_show_this_message_again" xml:space="preserve">
```

Comment:
Do_not

Reply:
fixed

Reply:
Does not look like this has actually been fixed.

Reply:
fixed for real this time :)